pylintrc

```diff
@@ -154,6 +154,8 @@ disable=print-statement,
         deprecated-sys-function,
         exception-escape,
         comprehension-escape,
+        too-many-arguments,
```

Comment:
I think we should make a one-time exception in the command arguments rather than remove coverage everywhere (e.g. https://stackoverflow.com/questions/28829236/is-it-possible-to-ignore-one-single-specific-line-with-pylint). This is kind of a special case since it's an API that's exposed to users.

Reply:
@aoen I think it's doing exactly the same. It is not removing coverage of all such occurrences, instead it is just a while list of skippable offenses. For ex these were the errors I was getting before
```
airflow/contrib/operators/hive_to_dynamodb.py:65:    def __init__(  # pylint:disable=too-many-arguments
airflow/operators/hive_to_druid.py:87:    def __init__(  # pylint:disable=too-many-arguments
airflow/operators/mysql_to_hive.py:88:    def __init__(  # pylint:disable=too-many-arguments
airflow/operators/s3_to_hive_operator.py:40:class S3ToHiveTransfer(BaseOperator):  # pylint:disable=too-many-instance-attributes
airflow/operators/s3_to_hive_operator.py:111:    def __init__(  # pylint:disable=too-many-arguments
```

So u can see that even #pylint was mentioned the error was being thrown.

Reply:
Sorry think I'm misunderstanding something! Do you mean the pylint comments were not working before?